priv/perf/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM elixir:1.10.4-alpine
+
+RUN apk add git curl bash maven jq \
+        autoconf \
+        automake \
+        gmp \
+        gmp-dev \
+        libtool \
+        gcc \
+        cmake \
+        gnupg \
+        alpine-sdk
+
+RUN git clone https://github.com/omgnetwork/elixir-omg
```

Comment:
I think you should be directly copy from the repo itself instead though. like: https://github.com/omgnetwork/elixir-omg/blob/c6cb81f063e2c4ffa0ef65da4f00406f9fa7fd2a/Dockerfile.watcher#L39

So you are not publishing current master but *this* current perf?

Reply:
instead of cloning the repo in the Dockerfile, I now copy the current elixir-omg directory https://github.com/omgnetwork/elixir-omg/pull/1753/files#diff-4a6ddb0f679098f7358db41ea45a306bc35e49328fc46afcd5a1e7bec2bf0130R14